src/adaptors/adaptor_listener.c

```diff
@@ -152,12 +150,9 @@ static void _listener_event_handler(pn_event_t *e, qd_server_t *qd_server, void
             if (li->on_accept)
                 li->on_accept(li, pn_event_listener(e), li->user_context);
             else {
-                // the adaptor_listener is closing but a connection attempt arrived before the
-                // cleanup is complete.  Need to accept it then close it when the connection
-                // completes
-                pn_raw_connection_t *close_me = pn_raw_connection();
-                pn_raw_connection_set_context(close_me, &_conn_event_context);
-                pn_listener_raw_accept(pn_event_listener(e), close_me);
+                // The adaptor_listener is closing but a connection attempt arrived before the
+                // cleanup is complete. Deny the connection attempt.
+                qd_adaptor_listener_deny_conn(li, pn_event_listener(e));
```

Comment:
Above there is a debug log message that says new incoming client connetion. When we explicitly add another log message when we deny the connection in the _deny_conn_handler ?

Reply:
Sure... coming up.

Reply:
Added with latest commit.